examples/raspberry-pi-pwm-loop.rs

```diff
@@ -0,0 +1,25 @@
+use navigator_rs::{pwm_Channel, Navigator};
+use rand::Rng;
+use std::thread::sleep;
+use std::time::Duration;
+
+fn main() {
+    println!("Creating your navigator module!");
+    let mut nav = Navigator::new();
+
+    println!("Setting up your navigator, ahoy!");
+    nav.init();
+
+    loop {
+        nav.set_pwm_value(pwm_Channel::C0, 1024);
+
+        nav.set_pwm_value(pwm_Channel::All, 2048);
+
+        let channels = &[pwm_Channel::C0, pwm_Channel::C1, pwm_Channel::C2];
+        let value: u16 = rand::thread_rng().gen_range(0..=4095);
+        nav.set_pwm_value_mult(Some(channels), value);
```

Comment:
I don't understand the logic behind this loop and how that is useful for the user as example.

The first line set C0 to 1024
The second line set all, including C0, to 204
The next line will change C0, and other two to a random number.

So... Why we have the first 3 lines ?

This can be a bit dangerous if someone connect a motor or something on the PWM as well, so I would recommend to not add this example at all.

Reply:
> the example cover 3 ways to set pwm. -> Individual channel -> All channels -> List of channels
> 
> maybe decrease everything to < 300 ? And use a different set of channels on the list

Logically the code does not make sense as I have previously pointed. For the functions available the user can check the documentation.

Reply:
We should only have the minimum necessary, other than that it will be harder to maintain or confused for the user.

Reply:
Please check new example,
reduced to 100 on pwm channels for safety reasons,
change c0,
change list [c1,c2,c3]
wait 10s and use channel "all" to turn off.

with this example user can fast manage pwm's use.